src/authgss_hash.c

```diff
@@ -92,17 +92,13 @@ svc_rpc_gss_cmpf(const struct opr_rbtree_node *lhs,
 	return (1);
 }
 
-void
+static void
 authgss_hash_init()
 {
 	int ix, code = 0;
 
 	mutex_lock(&authgss_hash_st.lock);
 
-	/* once */
-	if (authgss_hash_st.initialized)
-		goto unlock;
-
```

Comment:
No we don't.
Those checks are duplicates and are executed from the macro 
cond_init_authgss_hash()
which is only used for this purpose.